packages/web-components/src/components/ic-popover-menu/ic-popover-menu.tsx

```diff
@@ -20,100 +20,96 @@ import { createPopper } from "@popperjs/core";
   },
 })
 export class PopoverMenu {
-  private anchorEl: HTMLElement;
   private ARIA_LABEL: string = "aria-label";
+  private anchorEl: HTMLElement;
   private backButton: HTMLIcMenuItemElement;
   private currentFocus: number;
   private popoverMenuEls: HTMLIcMenuItemElement[] = [];
 
   @Element() host: HTMLIcPopoverMenuElement;
 
-  @State() openingFromChild: boolean = false;
-  @State() openingFromParent: boolean = false;
-
   /**
    * The ID of the element the popover menu will anchor itself to. This is required unless the popover is a submenu.
    */
   @Prop() anchor: string;
 
   /**
-   * @internal The parent popover menu of a child popover menu.
+   * The ID of the element that can open the popover menu
+   * if mouseAnchor is set to true. If null, then use the root element.
    */
-  @Prop() parentLabel?: string;
+  @Prop() clickEventEle: string = document.documentElement.id;
 
   /**
-   * @internal The parent popover menu of a child popover menu.
+   *  absolute screen X co-ordinate for popover menu render position.
    */
-  @Prop() parentPopover?: HTMLIcPopoverMenuElement;
+  @Prop() renderX: number = null;
+  /**
+   *  absolute screen Y co-ordinate for popover menu render position.
+   */
+  @Prop() renderY: number = null;
 
   /**
-   * The unique identifier for a popover submenu.
+   * The MouseEvent.button values that can trigger an ic-popover-menu open.
+   * Defaults to [2], i.e. mouse right click only
+   *
+   * 0 = left click
+   * 1 = middle click
+   * 2 = right click
+   * 3 = mouse browser back
+   * 4 = mouse browser forward
```

Comment:
Might be worth exporting these as an enum and setting the type to that as we only want to accept these numbers, not any number, and they're for the equivalent close prop